src/include/connection.h

```diff
@@ -618,6 +618,14 @@ struct __wt_connection_impl {
     /* AUTOMATIC FLAG VALUE GENERATION STOP 8 */
     uint8_t json_output; /* Output event handler messages in JSON format. */
 
+/*
+ * Variable with flags for which subsystems the diagnostic generation stress has been requested.
+ */
+/* AUTOMATIC FLAG VALUE GENERATION START 0 */
+#define WT_GEN_STRESS_SPLIT 0x1u
+    /* AUTOMATIC FLAG VALUE GENERATION STOP 8 */
+    uint8_t generation_stress_flags;
```

Comment:
My opinion would to place this under the debug flags and to seperate this when we have enough generation stress type flags. what are your thoughts @etienneptl ?